conjure-java/src/main/java/com/palantir/conjure/java/cli/ConjureJavaCli.java

```diff
@@ -90,16 +90,6 @@ public void run() {
                 description = "Generate retrofit interfaces for streaming/async clients")
         private boolean generateRetrofit;
 
-        @CommandLine.Option(names = "--retrofitCompletableFutures",
-                defaultValue = "false",
-                description = "Generate retrofit services which return Java8 CompletableFuture instead of OkHttp Call")
-        private boolean retrofitCompletableFutures;
-
-        @CommandLine.Option(names = "--retrofitListenableFutures",
```

Comment:
keep this otherwise we break 200 consumers

Reply:
I think we allow unknown arguments for this reason, removal shouldn't break anything